colibre/auto_plotter/metallicity.yml

```diff
@@ -507,17 +507,18 @@ stellar_mass_star_metallicity_50:
       units: "dimensionless"
   metadata:
     title: "Stellar mass - Star metallicity relation (50 kpc aperture)"
-    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126).
+    caption: Average stellar metallicity measured in the same aperture as the stellar mass. Gallazzi data is corrected from their choice of solar metallicity (0.02) to ours (0.0126). The Yates+ data from MaNGA corresponds to mass-weighted stellar metallicity within 3 arcsec for star-forming galaxies at 𝑧 = 0.
     section: Stellar Metallicity
     show_on_webpage: true
   observational_data:
     - filename: GalaxyStellarMassStellarMetallicity/Gallazzi2005_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Kirby2013_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5
-    - filename: GalaxyStellarMassStellarMetallicity/FIREbox.hdf5
```

Comment:
If you'd like to remove FIREbox, can we please discuss it separately and then possibly add these changes as part of another pull request?

The current pull request is supposed to do only one thing: add the new data, not remove the existing one.